packages/cdktf/lib/backends/local-backend.ts

```diff
@@ -49,13 +49,22 @@ export class DataTerraformRemoteStateLocal extends TerraformRemoteState {
     super(scope, id, "local", config);
   }
 }
-
+/**
+ * The local backend stores state on the local filesystem, 
+ * locks that state using system APIs, and performs operations locally.
+ * 
+ * Read more about this backend in the Terraform docs:
+ * https://www.terraform.io/language/settings/backends/http
+ */
 export interface LocalBackendProps {
   /**
-   * Path where the state file is stored.
-   * @default - defaults to terraform.${stackId}.tfstate
+   * (Optional) The path to the tfstate file. 
+   * This defaults to "terraform.tfstate" relative to the root module by default.
```

Comment:
While this is true for Terraform we have a different behaviour in place in CDKTF because we need to support multiple stacks. So this comment should be left as it was.

Reply:
have made changes, as asked by u.

Reply:
Ah, sorry, I should've been a bit more clear: The previous comment was enough already – having both now with contradictory explanations is not correct either 🙈